server/src/sim.rs

```diff
@@ -31,29 +34,38 @@ pub struct Sim {
     entity_ids: FxHashMap<EntityId, Entity>,
     world: hecs::World,
     graph: Graph,
+    /// Voxel data that has been fetched from a savefile but not yet introduced to the graph
+    preloaded_voxel_data: FxHashMap<ChunkId, VoxelData>,
     spawns: Vec<Entity>,
     despawns: Vec<EntityId>,
     graph_entities: GraphEntities,
     dirty_nodes: FxHashSet<NodeId>,
+    dirty_voxel_nodes: FxHashSet<NodeId>,
```

Comment:
This field should be documented.

Reply:
Oh, I guess that was pretty obvious in retrospect. Oh well, no harm!

Reply:
I still agree with your original suggestion, since one could imagine encountering this code before knowing about how voxel data and entity data is separated in save files and being confused.